internal/cassandra.go

```diff
@@ -197,3 +204,25 @@ func (db *defaultDatabase) RemoveFromTagIndex(tagKey string, tagValue string, me
 		tagValue,
 	).Exec()
 }
+
+func (db *defaultDatabase) AddGraphiteMetric(metric api.GraphiteMetric) error {
+	return db.session.Query(
+		"INSERT INTO graphite_names (shard, graphite_name) VALUES (?, ?) USING TTL 90000", // Slightly more than 24 hours TTL
```

Comment:
It's now configurable, through the API config YAML as `database: graphite_metric_ttl: (number)`.